thirdparty/README.md

```diff
@@ -29,6 +29,18 @@ Files extracted from upstream source:
 - `LICENSE`
 
 
+## brotli
+
+- Upstream: https://github.com/google/brotli
+- Version: 1.0.9 (e61745a6b7add50d380cfd7d3883dd6c62fc2c71, 2020)
```

Comment:
We could update to latest Git commit, it seems like there have been a number of significant fixes and no news of any plans to release a maintenance update.